src/cookies.test.ts

```diff
@@ -0,0 +1,67 @@
+		expect(getCookie('s_vi')).toEqual(
+			'[CS]v1|29AB343C05013259-4000014980021C3D[CE]',
+		);
+	});
+
+	it('should be able to get country cookie', () => {
```

Comment:
is this testing anything that the example above isn't testing?

Reply:
Nope! the more advanced cookie above is covering what we need.